conda/base/context.py

```diff
@@ -609,6 +620,20 @@ def aggressive_update_packages(self):
         from ..models.match_spec import MatchSpec
         return tuple(MatchSpec(s) for s in self._aggressive_update_packages)
 
+    @memoizedproperty
+    def prerelease_allowed_specs(self):
+        from ..models.match_spec import MatchSpec
+        return tuple(MatchSpec(s) for s in context.prerelease_allowed_packages)
```

Comment:
We should probably de-duplicate items here by having the MatchSpec `merge`d. E.g. `MatchSpec.merge(*specs)`.

Reply:
Sure, why not. However, in practice I would expect this to only contain a list of simple package names without any versions.